frontend/src/metabase/dashboard/reducers.js

```diff
@@ -406,20 +392,26 @@ const missingActionParameters = handleActions(
       next: (state, payload) => null,
     },
   },
-  null,
+  INITIAL_DASHBOARD_STATE.missingActionParameters,
 );
 
-export default combineReducers({
-  dashboardId,
-  isEditing,
-  loadingControls,
-  dashboards,
-  dashcards,
-  dashcardData,
-  slowCards,
-  parameterValues,
-  loadingDashCards,
-  isAddParameterPopoverOpen,
-  sidebar,
-  missingActionParameters,
-});
+export default reduceReducers(
```

Comment:
`reduceReducers` allows both the reducer defined below by `combineReducers` and the `tabsReducer` to operate on the `dashboard` slice of the redux store. See [this StackOverflow thread](https://stackoverflow.com/questions/38652789/correct-usage-of-reduce-reducers) for more details.